x-pack/plugins/secops/server/lib/kpi_network/types.ts

```diff
@@ -21,3 +21,20 @@ export interface KpiNetworkHit extends SearchHit {
     };
   };
 }
+
+export interface KpiNetworkHeader {
```

Comment:
optional -> these types can become global types because everyone who would like to do a msearch will have to recreate the same types so why not put them global from the beginning

Reply:
Agree, I'll add more queries in `msearch` soon. it will be covered in my next PR.